angular.json

```diff
@@ -28,6 +28,7 @@
               "src/assets"
             ],
             "styles": [
+              "./node_modules/@angular/material/prebuilt-themes/purple-green.css",
```

Comment:
These addition is not being used, since we already have a custom theme applied in the stylees.scss, so please remove these changes.